src/Cake.Core/Constants.cs

```diff
@@ -27,5 +27,12 @@ public static class Paths
             public const string Addins = "Paths_Addins";
             public const string Modules = "Paths_Modules";
         }
+
+        public static class Cache
+        {
+            public const string Enabled = "Cache_Enabled";
+            public const string Path = "Cache_Path";
```

Comment:
This could, however, also be `Paths_Cache` - to have the paths in one place...

Reply:
@nils-a adjusted to `Settings.EnableScriptCache` and `Paths.Cache` so. command-line becomes  i.e

`--settings_enablescriptcache=true --paths_cache="C:/temp/cache"`